development/Menu.js

```diff
@@ -49,7 +49,7 @@ export default class Menu extends React.Component {
               position: 'bottom-right',
               timeOut: 10000,
               transitionIn: 'bounceInDown',
-              transitionOut: 'bounceOutUp'
+              transitionOut: 'bounceOutU'
```

Comment:
The idea behind this pr is to allow a user to use any css class, but close a popup in an invalid case as well